### ISSUE TITLE
Close button on social login

### ISSUE THREAD

**Issue by Alexintosh** [Alexintosh/Protokit#3]

**Comment by Saumik** [comment]
Yeah this is a very specific requirement. The login box which appears should have a close option. It is a vey good starter into ionic but this is a specific problem, please solve it in the earliest. Thanks in advance.
